### PR TITLE
fix: fix dependencies and imports

### PR DESCRIPTION
**Pull request by patogallaiovlabs**
## Description

- Update of 'react-navigation' dep.
- Fix import of '@react-native-async-storage'

### CODE REVIEW

File: android/app/src/main/java/com/rsk/rwallet/v2/MainApplication.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 import com.facebook.react.PackageList;
 import com.facebook.react.ReactApplication;
+import com.swmansion.reanimated.ReanimatedPackage;
```
Comment (martinmedina) [inline android/app/src/main/java/com/rsk/rwallet/v2/MainApplication.java:10]: Duplicated import

File: android/settings.gradle [hunk 2]
```diff
@@ -1,4 +1,8 @@
 rootProject.name = 'rwallet'
+include ':react-native-reanimated'
+project(':react-native-reanimated').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-reanimated/android')
+include ':react-native-reanimated'
```
Comment (martinmedina) [inline android/settings.gradle:4]: Duplicated

**Review by martinmedina — APPROVED** [review]
Looks good, I suggest cleaning those duplicated lines of code.

File: android/app/src/main/java/com/rsk/rwallet/v2/MainApplication.java (hunk 1 above)
Comment (patogallaiovlabs) [reply android/app/src/main/java/com/rsk/rwallet/v2/MainApplication.java:10]: nice catch!

File: android/settings.gradle (hunk 2 above)
Comment (patogallaiovlabs) [reply android/settings.gradle]: nice catch!

**Pull request merged**